index.bs

```diff
@@ -130,8 +130,8 @@ or a combination of both.
 
 This specification relies on several other underlying specifications.
 
-: HTML5
-:: The concept of <dfn for='web'>origin</dfn> and the <dfn>Navigator</dfn> interface are defined in [[!HTML5]].
+: HTML 5.1
```

Comment:
I would s/HTML 5.1/HTML/ here since the citation in the next line is clearly to HTML51.